src/auth/types/forgotPasswordTokenPayload.interface.ts

```diff
@@ -1,3 +1,3 @@
-export interface ForgotPasswordTokenPayload {
-  email: string;
-}
+// export interface ForgotPasswordTokenPayload {
```

Comment:
remove the commented code or even better the entire file

Reply:
✅